src/event_loop.rs

```diff
@@ -111,19 +112,23 @@ impl EventSource for WaylandSource {
             // quickly. Either it is slowed down or we are a spammer.
             // Should not really happen, if it does we do nothing and will flush again later
         }
-        Ok(())
+        Ok(PostAction::Continue)
```

Comment:
Due to the new semantics of calloop, for full correctness the whole body of this function needs to be wrapped En the callback of `self.fd.process_events()`.

Reply:
Ok, moved into `self.fd.process_events`.

Reply:
Are you ok with the changes or is there anything else that needs to be changed?